docroot/sites/all/modules/contrib/focal_point/js/focal_point.js

```diff
@@ -17,7 +17,7 @@
         // Set some variables for the different pieces at play.
         var $indicator = $(this);
         var $img = $(this).siblings('img');
-        var focalPointID = $(this).attr('id');
+        var focalPointID = Drupal.checkPlain($(this).attr('id'));
```

Comment:
This is the change from the security fix. See https://cgit.drupalcode.org/focal_point/commit/?id=0fd7cd1